asset/js/app.js

```diff
@@ -249,6 +249,9 @@
             getMerchantName: function(merchant_id) {
                 return ((merchant_id in merchants) && merchants[merchant_id].merchant_name) ? merchants[merchant_id].merchant_name : merchant_id.substring(0, 8);
             },
+            getMerchantColor: function(merchant_id) {
+                return (merchant_id in merchants) ? merchants[merchant_id].color : '#FFFFFF';
```

Comment:
Why white? That's pretty much transparent, isn't it?

Reply:
It is basically just a fail safe. This only happens if we have a merchant which the marketplace doesn't know, which should never happen.

Reply:
@Bouncner Fixed issue.